acceptance-tests/src/test/java/uk/gov/di/ipv/cri/passport/acceptance_tests/pages/PassportPageObject.java

```diff
@@ -799,4 +800,27 @@ boolean isWithinRange(LocalDateTime testDate) {
 
         return testDate.isBefore(nbfMax) && testDate.isAfter(nbfMin);
     }
+
+    public static LocalDate subtractMonthsFromCurrentDate(int monthsToSubtract) {
```

Comment:
This can be a private method without static as were not using it outside this class